__tests__/app.test.js

```diff
@@ -47,3 +47,60 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("should return an array of comments with correct properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
```

Comment:
It's vital we have a length assertion here before going into a forEach. Without that we could be receiving an empty array back, meaning the forEach never runs and we get an auto pass.